pynamodb/migration.py

```diff
@@ -79,15 +83,21 @@ def migrate_boolean_attributes(model_class,
                                             meant to trigger the code path in boto, to allow us to unit test that
                                             we are jumping through appropriate hoops handling the resulting
                                             failure and distinguishing it from other failures.
+    :param page_size: Passed along to the underlying 'page_size'. Page size of the scan to DynamoDB.
+    :param limit: Passed along to the underlying 'limit'. Used to limit the number of results returned.
+    :param back_off: Number of seconds to sleep when exceeding capacity.
```

Comment:
nit pick: 
IMHO, i think back_off is not a clear variable name. 
I personally suggest using something more descriptive such as `sec_sleep_when_exceed_cap` or anything which can clearly tell people what this variable is.